packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -202,16 +202,19 @@ export class Field extends PureComponent {
             ...events,
             onChange: onChange || noopFn
         };
-
+        // if button value is "none" do not disable
+        const disableButton = (!attr.value?.match('none') && isDisabled);
+        const isChecked = disableButton ? !isDisabled : null;
+        // shipping options have checked attr assigned so prioritize its value
+        attr.checked = attr.checked ? attr.checked : isChecked;
```

Comment:
destruct please attr to get checked. and maybe still makes a point to creating new const with checked value and then pass it into input as attribute 'checked'?

Reply:
done